tools/server/webui/src/utils/types.ts

```diff
@@ -40,6 +40,7 @@ export interface Message {
   type: 'text' | 'root';
   timestamp: number; // timestamp from Date.now()
   role: 'user' | 'assistant' | 'system';
+  reasoningContent?: string;
```

Comment:
We cannot store reasoning content inside the message object - it must be contained inside `content`. Otherwise, it's impossible to modify assistant message (a planned feature)